tools/SWTbot-Testing/src/swtbot_testing/MavenInterface.java

```diff
@@ -0,0 +1,58 @@
+package swtbot_testing;
+
+import org.eclipse.swtbot.eclipse.finder.SWTWorkbenchBot;
+
+public interface MavenInterface
+{
+    static SWTWorkbenchBot bot =new SWTWorkbenchBot();
```

Comment:
It's good to set this common code outside Java test class, but usually we don't put codes like this in a interface.